server/api/controllers/liveSermon/scheduler-live-sermon.js

```diff
@@ -0,0 +1,60 @@
+const cron  = require('node-cron');
```

Comment:
have you tested if this whole logic works? i don't see anywhere where we're calling the cron job to start

Reply:
also, quite a loaded comment, but i don't think this should be the way we implement this. we already have a cron job running for caching (take a look at `server/config/bootstrap.js`). We should just be extending that

Reply:
lmk if you want clarfication

Reply:
actually, this is making it way too complicated. why do we need a cron job? why can't we just set LiveSermon object to have a `StartTime` and `EndTime` that will up and down according to the times?